packages/amplication-git-utils/src/git/github.service.ts

```diff
@@ -284,20 +302,18 @@ export class GithubService implements GitProvider {
     return null;
   }
 
-  async createPullRequest(
-    createPullRequestArgs: CreatePullRequestArgs,
-    files: Required<Changes["files"]>
+  async createPullRequestFromFiles(
+    createPullRequestFromFiles: CreatePullRequestFromFiles
   ): Promise<string> {
     const {
-      pullRequestMode,
       owner,
       repositoryName,
-      commit,
       pullRequestTitle,
       pullRequestBody,
-      head,
-    } = createPullRequestArgs;
-
+      branchName,
+      files,
+      commitMessage,
+    } = createPullRequestFromFiles;
     const myOctokit = Octokit.plugin(createPullRequest);
```

Comment:
add comment "We are not using this.octokit, instead we are using a local octokit client because we need the plugin"

Reply:
I added the comment